### ISSUE TITLE
Can I remove the image from my host?

### ISSUE THREAD

**Issue by bahramb92** [andrieslouw/imagesweserv#436]
Can I remove the image from my host after the service cached it to reduce the storage in use?

**Comment by andrieslouw** [comment]
No, unfortunately we need the origin to remain accessible, see also https://images.weserv.nl/faq/#how-are-images-cached-by-your-servers

**Comment by kleisauke** [comment]
I hope this information helped. Please feel free to re-open if questions remain.
